fun will_throw
{
 try {
  [ will_throw()... ]
}
catch const e {}

try {[ will_throw()... ]
}
catch const e {
  e++e
}yield 34
}var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
}
assert exc
